diff -ur xsd-4.0.0.orig/makefile xsd-4.0.0/makefile
--- xsd-4.0.0.orig/makefile	2014-07-21 09:36:38.000000000 +0300
+++ xsd-4.0.0/makefile	2015-04-28 18:40:16.104185101 +0300
@@ -15,8 +15,6 @@
 .PHONY: $(dist-pre)
 
 $(default): $(out_base)/xsd/      \
-            $(out_base)/tests/    \
-            $(out_base)/examples/ \
             $(out_base)/doc/
 
 # Test.
@@ -28,7 +26,6 @@
 #
 $(install): $(out_base)/xsd/.install           \
             $(out_base)/libxsd/.install        \
-	    $(out_base)/examples/.install      \
             $(out_base)/doc/.install
 	$(call install-dir,$(src_base)/dist/examples/build,$(install_doc_dir)/xsd/examples/build)
 	$(call install-dir,$(src_base)/dist/examples/cxx,$(install_doc_dir)/xsd/examples/cxx,-name makefile)
@@ -44,7 +41,6 @@
 #
 $(dist): $(out_base)/xsd/.dist           \
          $(out_base)/libxsd/.dist        \
-         $(out_base)/examples/.dist      \
          $(out_base)/doc/.dist
 	$(call install-dir,$(src_base)/dist/examples/build,$(dist_prefix)/examples/build)
 	$(call install-dir,$(src_base)/dist/examples/cxx,$(dist_prefix)/examples/cxx,-name makefile)
@@ -59,7 +55,6 @@
 $(dist-win): $(dist-pre)                         \
              $(out_base)/xsd/.dist-win           \
              $(out_base)/libxsd/.dist-win        \
-             $(out_base)/examples/.dist-win      \
              $(out_base)/doc/.dist-win
 	$(call install-dir,$(src_base)/dist/etc,$(dist_prefix)/etc)
 	$(call install-dir,$(src_base)/dist/examples/build,$(dist_prefix)/examples/build)
@@ -90,8 +85,6 @@
 # Clean.
 #
 $(clean): $(out_base)/xsd/.clean      \
-          $(out_base)/tests/.clean    \
-          $(out_base)/examples/.clean \
           $(out_base)/doc/.clean
 
 $(call include,$(bld_root)/install.make)
