diff --git a/host-linux/chrome-token-signing.pro b/host-linux/chrome-token-signing.pro
index f24f580..b563ff5 100644
--- a/host-linux/chrome-token-signing.pro
+++ b/host-linux/chrome-token-signing.pro
@@ -25,4 +25,4 @@ extension.path = /opt/google/chrome/extensions
 extension.files += ../ckjefchnfjhjfedoccjbhjpbncimppeg.json
 ffextension.path = /usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}
 ffextension.files += ../{443830f0-1fff-4f9a-aa1e-444bafbc7319}.xpi
-INSTALLS += target hostconf ffconf extension ffextension
+INSTALLS += target
