diff --git a/src/MainWindow.cpp b/src/MainWindow.cpp
index 3bf9b5b..117c97e 100644
--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -54,6 +54,7 @@ class MacMenuBar;
 #include <QtGui/QPainter>
 #include <QtGui/QPaintEvent>
 #include <QtNetwork/QSslKey>
+#include <QtWidgets/QButtonGroup>
 #include <QtWidgets/QFileDialog>
 #include <QtWidgets/QMessageBox>
 
diff --git a/src/Updater.cpp b/src/Updater.cpp
index 37f687b..e868b61 100644
--- a/src/Updater.cpp
+++ b/src/Updater.cpp
@@ -38,6 +38,7 @@
 #include <QtNetwork/QNetworkReply>
 #include <QtNetwork/QSslKey>
 #include <QtGui/QPainter>
+#include <QtGui/QRegExpValidator>
 #include <QtWidgets/QPushButton>
 
 #include <openssl/evp.h>
