--- a/CMakeLists.txt	2024-12-09 10:14:24.000000000 +0200
+++ b/CMakeLists.txt	2025-02-09 12:27:30.143422563 +0200
@@ -13,8 +13,8 @@
 
 find_package(LibDigiDocpp 4.1.0 REQUIRED)
 find_package( LDAP REQUIRED )
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} 5.12.0 REQUIRED COMPONENTS Core Widgets Network PrintSupport Svg LinguistTools)
+find_package(QT NAMES Qt6 COMPONENTS Core REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} 6.0.0 REQUIRED COMPONENTS Core Widgets Network PrintSupport Svg LinguistTools)
 find_package(FlatBuffers CONFIG REQUIRED NAMES FlatBuffers Flatbuffers)
 find_package(ZLIB REQUIRED)
 
--- a/common/CMakeLists.txt	2025-02-09 12:28:18.383837515 +0200
+++ b/common/CMakeLists.txt	2025-02-09 12:27:57.320322998 +0200
@@ -13,8 +13,8 @@
 endif()
 set_env( CONFIG_URL "https://id.eesti.ee/config.json" CACHE STRING "Set Config URL" )
 find_package(OpenSSL 1.1.1 REQUIRED)
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} 5.12.0 REQUIRED COMPONENTS Widgets Network LinguistTools)
+find_package(QT NAMES Qt6 COMPONENTS Core REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} 6.0.0 REQUIRED COMPONENTS Widgets Network LinguistTools)
 
 if(${QT_VERSION} VERSION_LESS "5.15.0")
 	macro(qt_add_translation)
