diff --git a/client/CMakeLists.txt b/client/CMakeLists.txt
index ae59c2e..e8598f9 100644
--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -1,13 +1,5 @@
-add_executable( TSLDownload TSLDownload.cpp )
-target_link_libraries(TSLDownload Qt${QT_VERSION_MAJOR}::Network)
 get_target_property(qtCore_install_prefix Qt${QT_VERSION_MAJOR}::qmake IMPORTED_LOCATION)
 get_filename_component(qtCore_install_prefix ${qtCore_install_prefix} DIRECTORY)
-add_custom_command(
-	OUTPUT TSL.qrc
-	DEPENDS TSLDownload
-	COMMAND $<TARGET_FILE:TSLDownload> "${CMAKE_CURRENT_BINARY_DIR}" ${TSL_URL} ${TSL_INCLUDE}
-	WORKING_DIRECTORY ${qtCore_install_prefix}
-)
 
 configure_file( translations/tr.qrc tr.qrc COPYONLY )
 qt_add_translation(SOURCES translations/en.ts translations/et.ts translations/ru.ts)
@@ -18,7 +10,7 @@ add_executable(${PROJECT_NAME} WIN32 MACOSX_BUNDLE
 	images/images.qrc
 	fonts/fonts.qrc
 	${CMAKE_CURRENT_BINARY_DIR}/tr.qrc
-	${CMAKE_CURRENT_BINARY_DIR}/TSL.qrc
+	TSL.qrc
 	main.cpp
 	Application.cpp
 	CheckConnection.cpp
Submodule common contains modified content
diff --git a/common/CMakeLists.txt b/common/CMakeLists.txt
index cb58b65..e1a30e6 100644
--- a/common/CMakeLists.txt
+++ b/common/CMakeLists.txt
@@ -47,13 +47,7 @@ if( CONFIG_URL )
 	if( LAST_CHECK_DAYS )
 		set_source_files_properties(Configuration.cpp PROPERTIES COMPILE_DEFINITIONS LAST_CHECK_DAYS=${LAST_CHECK_DAYS})
 	endif()
-	file( DOWNLOAD ${CONFIG_URL} ${CMAKE_CURRENT_BINARY_DIR}/config.json )
-	string( REPLACE ".json" ".rsa" RSA_URL ${CONFIG_URL} )
-	file( DOWNLOAD ${RSA_URL} ${CMAKE_CURRENT_BINARY_DIR}/config.rsa )
-	string( REPLACE ".json" ".pub" PUB_URL ${CONFIG_URL} )
-	file( DOWNLOAD ${PUB_URL} ${CMAKE_CURRENT_BINARY_DIR}/config.pub )
-	configure_file( config.qrc config.qrc COPYONLY )
-	qt_add_resources(CONFIG_SOURCES ${CMAKE_CURRENT_BINARY_DIR}/config.qrc)
+	qt_add_resources(CONFIG_SOURCES config.qrc)
 	target_compile_definitions(qdigidoccommon PUBLIC CONFIG_URL="${CONFIG_URL}")
 	target_sources(qdigidoccommon PRIVATE ${CONFIG_SOURCES} Configuration.cpp )
 endif()
