diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4b6ed04..0d6018b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -93,7 +93,7 @@ foreach(TSL_CERT ${TSL_CERTS})
     else()
         add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${BASE}.h
             COMMAND ln -sf ${TSL_CERT} ${BASE}.crt
-            COMMAND xxd -i ${BASE}.crt ${BASE}.h
+            COMMAND xxdi.pl ${BASE}.crt > ${BASE}.h
             COMMENT "Generating ${BASE}.h from ${TSL_CERT}"
             )
     endif()
