Avoid excessive dependency on app-editors/vim-core

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -119,7 +119,7 @@
     else()
         add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${BASE}.h
             COMMAND ln -sf ${TSL_CERT} ${BASE}.crt
-            COMMAND xxd -i ${BASE}.crt ${BASE}.h
+            COMMAND xxdi.pl ${BASE}.crt > ${BASE}.h
             COMMENT "Generating ${BASE}.h from ${TSL_CERT}"
             )
     endif()
